boards/renesas/rzg3s_smarc/doc/index.rst

```diff
@@ -0,0 +1,250 @@
+.. _rzg3s_smarc:
+
+RZ/G3S SMARC Evaluation Board Kit
+#################################
+
+Overview
+********
+
+The Renesas RZ/G3S SMARC Evaluation Board Kit (RZ/G3S-EVKIT) consists of a SMARC v2.1 module board and a carrier board.
+
+* Device: RZ/G3S R9A08G045S33GBG
+
+  * Cortex-A55 Single, Cortex-M33 x 2
+  * BGA 359-pin, 14mmSq body, 0.5mm pitch
+
+* SMARC v2.1 Module Board Functions
+
+  * LPDDR4 SDRAM: 1GB x 1pc
+  * QSPI flash memory: 128Mb x 1pc
+  * eMMC memory: 64GB x 1pc
+  * PMIC power supply RAA215300A2GNP#HA3 implemented
+  * microSD card x2
+  * I3C connector
+  * JTAG connector
+  * ADC x8 channels
+  * Current monitor (USB Micro B)
+
+* Carrier Board Functions
+
+  * Gigabit Ethernet x2
+  * USB2.0 x2ch (OTG x1ch, Host x1ch)
+  * CAN-FD x2
+  * microSD card x1
+  * Mono speaker, Stereo headphone, Mic., and Aux..
+  * PMOD x2
+  * USB-Type C for power input
+  * PCIe Gen2 4-lane slot (G3S supports only 1-lane)
+  * M.2 Key E
+  * M.2 Key B and SIM card
+  * Coin cell battery holder (3.0V support)
+
+.. figure:: rzg3s_smarc.webp
+	:align: center
+	:alt: RZ/G3S SMARC Evaluation Board Kit
+
+Hardware
+********
+
+The Renesas RZ/G3S MPU documentation can be found at `RZ/G3S Group Website`_
+
+.. figure:: rzg3s_block_diagram.webp
+	:width: 600px
+	:align: center
+	:alt: RZ/G3S group feature
+
+	RZ/G3S block diagram (Credit: Renesas Electronics Corporation)
+
+Supported Features
+==================
+
+The Renesas ``rzg3s_smarc/r9a08g045s33gbg/cm33`` board target supports for ARM Cortex-M33 System Core without FPU
+and the following hardware features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | arch/arm                            |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | arch/arm                            |
++-----------+------------+-------------------------------------+
+| PINCTRL   | on-chip    | pinctrl                             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial                              |
++-----------+------------+-------------------------------------+
+
+Other hardware features are currently not supported by the port.
+
+Programming and Debugging
+*************************
+
+RZ/G3S-EVKIT is designed to start different systems on cores.
```

Comment:
```suggestion
RZ/G3S-EVKIT is designed to start different systems on different cores.
```
(I think?)

Reply:
That's sound better. Thank you.
I updated as your suggestion.